lib/tasks/migrate_site_banners.rake

```diff
@@ -0,0 +1,18 @@
+namespace :site_banners do
+  task :migrate_site_banners => :environment do
```

Comment:
would this be better under the existing `database_updates` namespace?

Reply:
Done ✔️